modules/update-service-delete-service-web-console.adoc

```diff
@@ -0,0 +1,18 @@
+[id="update-service-delete-service-web-console_{context}"]
+= Deleting an OpenShift Update Service application by using the web console
+
+You can use the {product-title} web console to delete an OpenShift Update Service application by using the OpenShift Update Service Operator.
+
+.Prerequisites
+
+* The OpenShift Update Service Operator has already been installed.
+
+.Procedure
+
+To delete an OpenShift Update Service application by using the {product-title} web console:
+
+. In the {product-title} web console, click *Operators* -> *Installed Operators*.
+
+. Click the *Update Service* tab.
+
+. From the list of installed OpenShift Update Service applications, select the menu icon at the far right of the application to be deleted and then click *Delete UpdateService*.
```

Comment:
A `Delete UpdateService` alert pops up, need to click `Delete` to proceed the deletion.
![Screenshot from 2021-03-25 15-46-32](https://user-images.githubusercontent.com/22511421/112437366-d643d780-8d81-11eb-8184-a9f485b0108c.png)

Reply:
Fixed